documents/Code of Ethics.md

```diff
@@ -1,87 +1,97 @@
          2. Lack of equipment to hold an event
          3. Event frequency in the area
          4. Inexperience of the community
-         5. Inexperience of the organizer
-         6. Past problems with an event held by the organizer/community
-      2. Delegates should work alongside the organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
-   4. Delegates and Organizers should only be officially listed for a competition if they perform a necessary role in the running of the event.
+         5. Inexperience of the Organizer
+         6. Past problems with an event held by the Organizer/community
+      2. Delegates should work alongside the Organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
+   4. Delegates and Organizers should only be officially listed for a competition if they perform a significant role.
+      1. Delegates and Organizers should agree on their roles for the competition before the competition is announced.
 4. **Conflicts of Interest**
-   1. WCA Staff must report all potential conflicts of interest that may arise and affect their WCA Staff role. Conflicts of interest will be reported to the WEC. These include but are not limited to:
+   1. WCA Staff must report all potential conflicts of interest that may arise and affect their WCA Staff role to the WEC using [this form]{https://docs.google.com/forms/d/e/1FAIpQLSca81GIwjguJoWrPcbgabkRpdgJqbusIf9RBR7ObNNNL9kvqw/viewform?usp=sf_link} . These include, but are not limited to:
       1. Sponsorships
-      2. A stake in businesses trading within the community
+      2. A stake in businesses trading within the cubing community.
          1. Example: Being the owner of a cubing store/business.
-      3. Any personal profit gained by actions taken within their WCA Staff role
    2. WCA Staff must not receive sponsorship(s) for their role within the WCA.
-      1. Any sponsorship details of a WCA Staff member must be reported to the WEC.
-   3. WCA Staff must not promote a third party brand who are not the competition sponsor(s) or sponsors of the WCA, within their role in the WCA. This includes but is not limited to:
-      1. Wearing the apparel of a sponsor while acting within their WCA Staff role at a competition
-      2. Promotions in emails to competitors
-      3. References in the email signature block
-   4. WCA Delegates must not wear sponsorship apparel of a personal sponsor during a competition in which they are listed as a Delegate.
-      1. Exception: Listed Delegates may only wear their sponsorship apparel while they are competing at a competition.
+      1. Any sponsorship details of a WCA Staff member as a competitor must be reported to the WEC.
+   3. WCA Staff must not use their role to promote a third party brand who are not [a WCA-recognized regional organization](https://www.worldcubeassociation.org/organizations), the competition sponsor(s), or sponsors of the WCA. This includes but is not limited to:

```

Comment:
What about a not officially recognised "sub regional organisation"? I don't feel these should be excluded, and they should be encouraged, since they enable better management of competition funds in a manner compliant with the code of ethics!

Reply:
If they are not recognized then they haven't been "vetted" by the WCA so we have no basis for what standards they hold or what their Mission is etc. I also think it's naive that you feel there are no regional organizations who arrange their finances in a way that would not be compliant with the CoE.

Don't forget this covers all Staff, for example if a communication team member started making WCA posts to promote a non-recognized regional organization, I think you can agree it would be an inappropriate use of their role and access.

Reply:
I certainly don't think that just because it is an organisation it is compliant with the CoE, but I think that having things structured via a regional organisation is better than not doing so.